### PR TITLE
window: store event_mask using EventMask

### LINKED ISSUE #338: gdk::WindowAttr should contain an EventMask in the event_mask field
Currently the `event_mask` field has type `i32`. The `EventMask` struct exists and can be built as a bitfield, and then converted using `.bits()`, yielding an `u32` which then needs to be cast in order to be used in the `event_mask` field. This is somewhat unwieldy, and it would likely be preferable to just have the `WindowAttr` struct contain the `EventMask`.

### PR DESCRIPTION
**Pull request by gdesmott**
Fix #338

### CODE REVIEW

File: src/window.rs [hunk 1]
```diff
@@ -94,7 +96,7 @@ impl<'a> ToGlibPtr<'a, *mut gdk_sys::GdkWindowAttr> for WindowAttr {
 
         let mut attrs = Box::new(gdk_sys::GdkWindowAttr {
             title: title.0 as *mut c_char,
-            event_mask: self.event_mask,
+            event_mask: i32::try_from(self.event_mask.bits()).unwrap(),
```
Comment (sdroege) [inline src/window.rs:99]: This would fail if the top bit is set. Should just be a cast

Comment (gdesmott) [reply src/window.rs]: fixed.

**Comment by EPashkin** [comment]
👍

**Comment by sdroege** [comment]
@GuillaumeGomez Green CI :)

**Comment by GuillaumeGomez** [comment]
Thanks @gdesmott !

**Pull request merged**